pipenv/utils.py

```diff
@@ -661,8 +661,10 @@ def convert_deps_from_pip(dep):
             req.path = None
 
         # Crop off the git+, etc part.
-        dependency.setdefault(req.name, {}).update({req.vcs: req.uri[len(req.vcs) + 1:]})
-
+        if '+' in req.uri:
```

Comment:
So I don't know if the `+` check by itself will be sufficient. That's a legal character in the path as well. We need to determine if there's a + explicitly in the scheme.

Reply:
yeah good call